jobs/ftp-poller/utils/minio.py

```diff
@@ -0,0 +1,47 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module is a wrapper for Minio."""
+import io
+
+from flask import current_app
+from minio import Minio
+from urllib3 import HTTPResponse
+
+
+class MinioService:
```

Comment:
Since this is under utils, should we treat these as some util methods rather than a service ?

Reply:
made it as minioutil